docs/reference-implementation/Health-Failure-Analysis.md

```diff
@@ -36,7 +36,7 @@ Global replication protects Cosmos DB instances from regional outage. The Cosmos
 
 | **Risk**                   | **Impact/Mitigation/Comment**                | **Outage**                 |
 | -------------------------------------------- | ------------------------------------------------------------ | ------------------------------------------- |
-| **Database/collection is renamed**     | Can happen due to mismatch in configuration when deploying – Terraform would overwrite the whole database, which could result in data loss (this can be prevented by using [database/collection  level locks](https://feedback.azure.com/forums/263030-azure-cosmos-db/suggestions/35535298-enable-locks-at-database-and-collection-level-as-w)). <br />**Application will not be able to access any data**. App configuration needs to be updated and pods restarted. | Yes                     |
+| **Database/collection is renamed**     | Can happen due to mismatch in configuration when deploying – Terraform would overwrite the whole database, which could result in data loss (this can be prevented by using [database/collection  level locks](https://docs.microsoft.com/en-us/azure/cosmos-db/scripts/cli/sql/lock)). <br />**Application will not be able to access any data**. App configuration needs to be updated and pods restarted. | Yes                     |
```

Comment:
```suggestion
| **Database/collection is renamed**     | Can happen due to mismatch in configuration when deploying – Terraform would overwrite the whole database, which could result in data loss (this can be prevented by using [database/collection  level locks](https://docs.microsoft.com/azure/cosmos-db/scripts/cli/sql/lock)). <br />**Application will not be able to access any data**. App configuration needs to be updated and pods restarted. | Yes                     |
```